reporters_db/data/regexes.json

```diff
@@ -54,6 +54,8 @@
         "with_commas_and_suffix#": "Page number that allows internal commas, plus optional alpha character appended",
         "with_commas_or_periods": "(?P<page>\\d(?:[\\d,.]*\\d)?)",
         "with_commas_or_periods#": "Page number that allows internal punctuation",
+        "with_letter": "(?P<page>\\d+\\w)",
```

Comment:
I think this is too broad a brush to paint with.  looking at your examples, I only see A or B being used - so to give [a-zA-Z] to deal with an occasional A or B seems to loose.

Reply:
Yes, i'll limit the regex with [a-zA-Z]